lib/getMetafieldsForCheckout.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function () {
+  const metaFields = []
+
+  if (!window || !window._recart) {
+    return metaFields
+  }
+
+  if (window._recart.getSessionId && window._recart.getSessionId()) {
```

Comment:
I wanted to make sure that we 

- prevent TypeErrors
- don't add empty metafields to the checkout